keyids

```diff
@@ -3,3 +3,4 @@
 0x5FE92C12EE88E1F0 Robin Schneider <ypid>
 0x489A4D5EC353C98A Robin Schneider <ypid>
 0xDAA9DC5E750C1E85 Aleksey Gavrilov <le9i0nx>
+0x610CB25237B370E9EB2108E89CEE1B6B059B598E <muelli>
```

Comment:
Also, please check the source to see if full fingerprints instead of keyids are supported.